.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "restructuredtext.confPath": "${workspaceFolder}/docs"
+}
```

Comment:
I'm sure `.vscode/` is in the `.gitignore`. How did this get committed back?
Please try to not commit editor config. If we allow it, we may eventually end up with everyones editor config and having to handle those conflcts.